docs/reference/manifest/progid.md

```diff
@@ -0,0 +1,30 @@
+---
+title: ProgID element in the manifest file
+description: ''
+ms.date: 04/22/2019
+localization_priority: Normal
+---
+
+# ProgID element
+
+Specifies the programmatic identifier of the equivalent COM add-in for the task pane of your web add-in.
+
+**Add-in type:** Task pane
+
+## Syntax
+
+```XML
+    <ProgID>{progid}</ProgID>  
+```
```

Comment:
Based on [this page](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/manifest/providername), this should look like:
```
<ProgID>string</ProgID>  
```